client/src/utils/get-nav-height.ts

```diff
@@ -0,0 +1,6 @@
+import {Breakpoint} from "../amplify-ui/styles/media";
+
+export const getNavHeight = (rems?: boolean) => {
+  const value = innerWidth > Breakpoint.LAPTOP * 16 ? 3 : 6.25;
```

Comment:
Comment this

Reply:
Actually let's just add an ID to the navbar then query this id. We can then use the height of the element